src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -932,13 +910,23 @@ protected override void EndProcessing()
             {
                 // Load the source if they want to load from a file
                 if (String.Equals(ParameterSetName, "FromPath", StringComparison.OrdinalIgnoreCase) ||
-                    String.Equals(ParameterSetName, "FromLiteralPath", StringComparison.OrdinalIgnoreCase))
+                    String.Equals(ParameterSetName, "FromLiteralPath", StringComparison.OrdinalIgnoreCase)
+                   )
                 {
-                    this.sourceCode = "";
+                    StringBuilder sb = new StringBuilder(paths.Length);
+                    sourceCode = "";
```

Comment:
sourceCode doesn't need to be initialized since it is always assigned after loop.  Also we normally name class fields with beginning underscore: "_sourceCode".

Reply:
I'll remove the line.

`sourceCode` is used in many places in the file. If I rename it will be very noisy and unrelated with the performance fix - I think it is better to leave it as is.